UnityProject/Assets/Scripts/Items/Medical/Implanter.cs

```diff
@@ -0,0 +1,137 @@
+		}
+	}
+
+	public bool WillInteract(HandApply interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side)) return false;
+		//can only be applied to LHB
+		if (!Validations.HasComponent<LivingHealthMasterBase>(interaction.TargetObject)) return false;
+
+		if (interaction.Intent != Intent.Help || primed == false) return false;
+
+		if(itemStorage.GetIndexedItemSlot(0).IsEmpty)
+		{
+			Chat.AddExamineMsgFromServer(interaction.Performer, "No implant present in implanter!");
+			return false;
+		}
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandApply interaction)
+	{
+		var LHB = interaction.TargetObject.GetComponent<LivingHealthMasterBase>();
+
+		foreach (var bodyPart in LHB.SurfaceBodyParts)
+		{
+			bool selected = false;
+
+			if (bodyPart.BodyPartType == interaction.TargetBodyPart)
+			{
+				ItemSlot toSlot = bodyPart.OrganStorage.GetNextFreeIndexedSlot();
+				ItemSlot fromSlot = itemStorage.GetIndexedItemSlot(0);
+
+				if (toSlot != null && selected == false)
+				{
+					selected = true;
+
+					ToolUtils.ServerUseToolWithActionMessages(interaction, timeToImplant,
+					$"You begin injecting {interaction.TargetObject.ExpensiveName()}'s {bodyPart.gameObject.ExpensiveName()} with the implanter...",
+					$"{interaction.Performer.ExpensiveName()} begins injecting {interaction.TargetObject.ExpensiveName()}'s {bodyPart.gameObject.ExpensiveName()} with the implanter...",
+					$"You inject {interaction.TargetObject.ExpensiveName()}'s {bodyPart.gameObject.ExpensiveName()} with the implanter.",
+					$"{interaction.Performer.ExpensiveName()} injects {interaction.TargetObject.ExpensiveName()}'s {bodyPart.gameObject.ExpensiveName()} with the implanter.",
+					() =>
+					{
+						Inventory.ServerTransfer(fromSlot, toSlot,ReplacementStrategy.DropOther);
+						
+						TogglePrimed();	
+					});
+					
+				}
+			}
+		}
+	}
+
+	public bool WillInteract(InventoryApply interaction, NetworkSide side)
+	{
+		if (DefaultWillInteract.Default(interaction, side) == false) return false;
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(InventoryApply interaction)
+	{
+		if (interaction.TargetObject == gameObject && interaction.IsFromHandSlot)
+		{
+			if (interaction.UsedObject != null)
+			{
+				Inventory.ServerTransfer(interaction.FromSlot, itemStorage.GetIndexedItemSlot(0));
+			}
+			else
+			{
+				Inventory.ServerTransfer(itemStorage.GetIndexedItemSlot(0), interaction.FromSlot);
+			}
+		}
+	}
+
+	public bool WillInteract(HandActivate interaction, NetworkSide side)
+	{
+		if(DefaultWillInteract.Default(interaction, side) == false) return false;
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandActivate interaction)
+	{
+		if (interaction.IsAltClick)
+		{
+			itemStorage.ServerDropAll();
+		}
+		else
+		{
+			if (itemStorage.GetIndexedItemSlot(0).IsEmpty) Chat.AddExamineMsg(interaction.Performer, "Cannot prime without implant");
```

Comment:
separate the body of the if into its own line and enclose it with curly brackets.